### DISCUSSION TITLE
Windows 11: Instanz läuft anscheinen 2x

### DISCUSSION THREAD

**Discussion by wgreipl** [localsend/localsend#858] [L-Deutsch] [answered]
Hey Community,

ich bekomme auf den sendenden Geräten meinen Windows 11-Rechner 2-fach in der Liste.

Im Empfangsmodus sieht man auch das wohl zwei Kanäle geöffnet wurden.

Funktionieren tun ja beide, sieht aber doof aus ;-)

![2023-11-02 19_27_51-LocalSend](https://github.com/localsend/localsend/assets/4572169/48518b8d-584f-4792-bce1-5169894d5539)

**Comment by Tienisto** [comment, marked as answer]
Anscheinend hat der Windows zwei Netzwerkadapter, die gleichzeitig aktiv sind. Auf dem Info-Button stehen die genauen lokalen IP-Adressen

**Reply by wgreipl** [reply]
Absolut richtig.

Vielen Dank.
